selenium/javaScript/Chapter2:-nightwatch for Multiple browser/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "JS-Nightwatch.js",
+  "version": "0.0.0",
+  "description": "NightWatch examples for pcloudy",
+  "main": "",
+  "scripts": {
+    "test": "./node_modules/.bin/nightwatch -e chrome,firefox,edge,safari tests"
```

Comment:
create different script for linux and win and use only that env which are in nightwatch.json